Microsoft.Toolkit.Uwp.UI.Controls.Layout/ListDetailsView/ListDetailsView.Properties.cs

```diff
@@ -63,7 +62,7 @@ public partial class ListDetailsView
             nameof(ListHeader),
             typeof(object),
             typeof(ListDetailsView),
-            new PropertyMetadata(null, OnListHeaderChanged));
+            new PropertyMetadata(null));
```

Comment:
OnListHeaderChanged is needed to collapse the content space if the header is blank. It also ensures it is shown if going from blank to something. Especially with the Lazy nature of it

Reply:
Done.